Looks like tests might need a --force or --noinput type of argument.
```
Uninstalling jsonobject-0.7.1:
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/DESCRIPTION.rst
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/INSTALLER
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/METADATA
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/RECORD
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/WHEEL
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/metadata.json
  /vendor/lib/python2.7/site-packages/jsonobject-0.7.1.dist-info/top_level.txt
  /vendor/lib/python2.7/site-packages/jsonobject/__init__.py
  /vendor/lib/python2.7/site-packages/jsonobject/__init__.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/api.py
  /vendor/lib/python2.7/site-packages/jsonobject/api.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/base.py
  /vendor/lib/python2.7/site-packages/jsonobject/base.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/base_properties.py
  /vendor/lib/python2.7/site-packages/jsonobject/base_properties.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/containers.py
  /vendor/lib/python2.7/site-packages/jsonobject/containers.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/exceptions.py
  /vendor/lib/python2.7/site-packages/jsonobject/exceptions.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/properties.py
  /vendor/lib/python2.7/site-packages/jsonobject/properties.pyc
  /vendor/lib/python2.7/site-packages/jsonobject/utils.py
  /vendor/lib/python2.7/site-packages/jsonobject/utils.pyc
Proceed (y/n)? Exception:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/pip/basecommand.py", line 209, in main
    status = self.run(options, args)
  File "/usr/local/lib/python2.7/site-packages/pip/commands/uninstall.py", line 76, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/usr/local/lib/python2.7/site-packages/pip/req/req_set.py", line 326, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/usr/local/lib/python2.7/site-packages/pip/req/req_install.py", line 746, in uninstall
    paths_to_remove.remove(auto_confirm)
  File "/usr/local/lib/python2.7/site-packages/pip/req/req_uninstall.py", line 103, in remove
    response = ask('Proceed (y/n)? ', ('y', 'n'))
  File "/usr/local/lib/python2.7/site-packages/pip/utils/__init__.py", line 158, in ask
    response = input(message)
EOFError: EOF when reading a line
```

--- a/scripts/uninstall-requirements.sh
+++ b/scripts/uninstall-requirements.sh
@@ -6,7 +6,7 @@ set -ev
 # because of a bug in how we were packaging jsonobject-couchdbkit earlier
 # (the `jsonobject/` directory was being packaged in with jsonobject-couchdbkit inappropriately)
 # feel free to delete this workaround after May 2016 (to be conservative)
-pip freeze | grep jsonobject-couchdbkit==0.7.2.0 && pip uninstall jsonobject
+pip freeze | grep jsonobject-couchdbkit==0.7.2.0 && pip uninstall jsonobject --yes
 
 uninstall=requirements/uninstall-requirements.txt
 tmp=requirements/uninstall-tmp.txt

